src/test/java/com/courtcanva/ccfranchise/repositories/FranchiseeRepositoryTest.java

```diff
@@ -44,6 +52,6 @@ public void shouldReturnFranchiseeWhenFranchiseeIdIsExist() {
         Franchisee franchisee = FranchiseeTestHelper.createFranchisee();
         franchiseeRepository.save(franchisee);
 
-        assertEquals(franchisee.getAbn(), Objects.requireNonNull(franchiseeRepository.findFranchiseeById(franchiseeId).orElse(null)).getAbn());
+        assertEquals(franchisee.getAbn(), Objects.requireNonNull(franchiseeRepository.findFranchiseeById(franchisee.getId()).orElse(null)).getAbn());
```

Comment:
The result will always be true if you use franchisee.getId here

Reply:
I modified this line you wrote because in your version `franchiseeId==1`, `franchiseeRepository.findFranchiseeById(1)` works in h2 database but will fail in Postgresql.
Postgresql d

Reply:
But now you didn't use postgres, right?